LanguageFeatures/Patterns/constant_A06_t01.dart

```diff
@@ -61,6 +61,10 @@ main() {
     case C():
 //       ^^^
 // [analyzer] unspecified
+// [cfe] unspecified
+    case const(C().v):
```

Comment:
Yes!

Just one personal style nit (probably also enforced by `dart format`): You could say "we must aim to test all possible programs, not just the pretty ones", which is true, but I still prefer a style where keywords are marked as different from expressions denoting functions by having a space before '('. So it's `if (b) S` and `while (b) S`, not `if(b) S` and `while(b) S`; similarly `const (C().v)` rather than `const(C().v)`. So unless you really want to test that the lexer can handle `const(`, `const (` would be nice. ;-)